---
bool connection_check_result
